packages/app/README.md

```diff
@@ -1,3 +1,3 @@
-# @masknet/decrypt-spa
+# @masknet/app
```

Comment:
@guanbinrui renaming it to app looks confusing. does that means it will become a full-functional replacement of Mask extension?

Reply:
> renaming it to app looks confusing

Many web3 applications display a "Launch App" button for users, which redirects them to a specific website such as `https://app.x.com`. It will be `https://app.mask.io` for Mask. As I am using `app`, I will utilize this convention.

> does that means it will become a full-functional replacement of Mask extension?

Perhaps in the distant future, there could be a purely web-based version of Mask.